src/nix/store-gc.md

```diff
@@ -8,14 +8,20 @@ R""(
   # nix store gc
   ```
 
-* Delete up to 1 gigabyte of garbage:
+* Perform garbage collection if there is less than 1 GiB of free space
+  in `/nix/store`, and stop once there is at least 5 GiB of free
+  space.
 
   ```console
-  # nix store gc --max 1G
+  # nix store gc --gc-threshold 1G --gc-limit 5G
```

Comment:
```suggestion
  # nix store gc --gc-threshold 10G --gc-limit 30G
```

These numbers are more realistic, allowing for larger builds without consuming much space.